airflow/providers/google/cloud/sensors/bigquery.py

```diff
@@ -163,6 +165,9 @@ def __init__(
         self.delegate_to = delegate_to
         self.impersonation_chain = impersonation_chain
 
+        self.deferrable = deferrable
+        self.poke_interval = poke_interval
```

Comment:
BaseSensorOperator already has a `poke_interval` and this would break it. Also the `5` default is much too small for traditional sensors, so something like this is probably needed:

```python
def __init__(self, ..., **kwargs):
    try:
        poke_interval: int = kwargs.pop("poke_internal")
    except KeyError:
        poke_interval = 5 if deferrable else 60
    super().__init__(**kwargs, poke_interval=poke_interval)
```

It may be worthwhile to push this logic into BaseSensorOperator somehow.

Reply:
Thanks! I didn't notice it. I'll check again on all related PRs. But I'm a bit confused about why we want to push this logic into BaseSensorOperator?

Reply:
Also, I'm thinking of implementing it in the following way. What do you think?

```python
        super().__init__(**kwargs)

        if self.deferrable:
            self.poke_interval = kwargs.get("poke_interval", 5)
```

Reply:
I think this works as well but could be problematic if the logic in the base class changes (not likely but still).

How about this?

```python
if deferrable and "poke_interval" not in kwargs:
    kwargs["poke_interval"] = 5
super().__init__(**kwargs)
```

Reply:
Just pushed a new version